### DISCUSSION TITLE
보스에 대해서 얘기해보아요

### DISCUSSION THREAD

**Discussion by rlatldnd1** [YOON-JAEYEONG/Student_Survivor#20] [General]
보스의 움직임, 행동 조건, 공격 방식 등등 보스의 모든 것에 대해 토의하는 Discussion 입니다. 자유롭게 토의해주세요.
적은 쓰레드들을 기반으로 보스를 구현합니다

**Comment by rlatldnd1** [comment]
보스는 일단 몸집 크게. 체력 많이. 속도는 느리게
보스1은 수면공격
보스2는 압도적인 물량 공격

보스 나올 때 기본 적들(퀴즈, 과제, 시험) 나오는건 그대로

**Comment by YOON-JAEYEONG** [comment]
보스가 나타나면 HUD에 경험치 아래쪽에다가 보스 피 표시해주는 것도 좋을 것 같아요
